scripts/codemod/index.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import type {ESNode, CallExpression} from 'hermes-estree';
+import type {TransformContext} from 'hermes-transform';
+
+const {transform, t} = require('hermes-transform');
+const Glob = require('glob');
+const {readFileSync, writeFileSync} = require('fs');
+
+const prettierOptions = require('../../.prettierrc.js');
+// path based overrides don't work, this applies to most files
+prettierOptions.trailingComma = 'all';
```

Comment:
I believe you can do the following in each transform so path based overrides work:
```
const prettierOptions = await Prettier.resolveConfig(filePath, {
  config:'../../.prettierrc.js',
  editorconfig: false,
});
```